execute_two.py

```diff
@@ -24,18 +24,18 @@ def __init__(self, menu, plots=None):
 
 DirIn,field=parser.parse(sys.argv)
 
-Vradar = Variable(None)
+
 
 
 app = QtGui.QApplication(sys.argv)
 
-MainMenu = Menu(Vradar, DirIn, name="Menu") #initiate Vradar
+MainMenu = Menu(DirIn, name="Menu") #initiate Vradar
+Vradar = MainMenu.Vradar
 
 Vtilt = Variable(0)
 Vtilt2 = Variable(0)
-Vlims = Variable(None)
-plot1 = Display(Vradar, Variable(field), Vtilt, Vlims, name="Display1", parent=MainMenu)
-plot2 = Display(Vradar, Variable(field), Vtilt2, Vlims, name="Display2", parent=MainMenu)
+plot1 = Display(Vradar, Variable(field), Vtilt, name="Display1", parent=MainMenu)
+plot2 = Display(Vradar, Variable(field), Vtilt2, name="Display2", parent=MainMenu)
 
 app.exec_()
 
```

Comment:
This would be an interesting thing to have, however I tend to believe as complex and powerful a graphical interface could be, it will never be as powerful and versatile as text. My idea is that powerful user will still work with execute scripts, so as long as we can satisfies less powerful users with a standard execute there is no need to spend time in this.

That being said, I will create a graphical component to control how V variable are shared (issue #14). This could be extended in the future to handle the components it self.